### DISCUSSION TITLE
Where do I need to add Google SEO domain property

### DISCUSSION THREAD

**Discussion by 28development** [vercel/community#796] [Help] [answered]
Hi,

I am adding a domain property on Google Search console and now need to add a string 

_Copy the TXT record below into the DNS configuration for mydomain.com_

I am confused, where in the domain section on vercel do I need to add this?

**Comment by 28development** [comment, marked as answer]
Found the field needed to click on the option field to add a TXT

**Reply by amyegan** [reply]
Glad you found what you needed! 🥳 

Adding some more info for anyone else who has this problem in the future: You can get to your custom domain settings at https://vercel.com/dashboard/domains. From there, click the domain you want to modify and you will see a form to add a new DNS record.
